src/index.js

```diff
@@ -90,7 +90,7 @@ function createPlugin() {
       const possibleThemes = await getPossibleThemes(
         theme,
         await cache.get('themes'),
-        path.dirname(markdownNode.fileAbsolutePath),
+        markdownNode.fileAbsolutePath ? path.dirname(markdownNode.fileAbsolutePath) : null,
```

Comment:
```suggestion
        // Node could be sourced from something other than a File node
        markdownNode.fileAbsolutePath ? path.dirname(markdownNode.fileAbsolutePath) : undefined,
```

Reply:
If you can commit this suggestion I’ll go ahead and merge. The types for `getPossibleThemes` allow `undefined` but not `null` for `contextDirectory`. It currently type-checks either way because I don’t have --strictNullChecks enabled, but may turn it on eventually.

Reply:
Actually, I have another fix I want to roll this in with, so I’ll merge and make the changes afterward. Thanks!